compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Check.hs

```diff
@@ -128,6 +128,10 @@ kindOfBuiltin = \case
   BTArrow -> KStar `KArrow` KStar `KArrow` KStar
   BTAny -> KStar
   BTTypeRep -> KStar
+  BTAnyException -> KStar
```

Comment:
We need a strategy for making sure we don't put anything related to exceptions into old DAML-LF packages. I guess our primary approach will be to guard it with CPP macros and the conversion to DAML-LF. I' wondering if it would make sense to guard it here as well in order to have a sanity check. There's the `_checkFeature` function towards the bottom of this file, which could help. Maybe we should do this for other features as well, but then in a separate PR.

Reply:
That sounds reasonable.